assets/app/views/catalog/_image.html

```diff
@@ -0,0 +1,49 @@
+<div class="col-md-4">
+  <div class="tile tile-image tile-click label-tags" id="" title="">
+    <div class="tile-table">
+      <div class="tile-table-cell image-icon">
+        <span aria-hidden="true" class="font-icon {{imageRepo | iconClass : 'image' : (imageTag + '.iconClass')}}" title="{{imageRepo.metadata.name}}:{{imageTag}}"></span>
+      </div>
+      <div class="tile-table-cell">
+        <h3>
+        <a href="javascript:;" class="tile-target" data-toggle="modal" data-target="#{{imageRepo.metadata.namespace}}_{{imageRepo.metadata.name}}_{{imageTag}}">{{imageRepo.metadata.name}}:{{imageTag}}</a>
+        </h3>
+        <p ng-if="imageRepo | annotation : (imageTag + '.description')">{{imageRepo | annotation : (imageTag + '.description')}}</p>
+      </div>
+    </div>
+  </div>
+  <div class="modal modal-create fade" id="{{imageRepo.metadata.namespace}}_{{imageRepo.metadata.name}}_{{imageTag}}" tabindex="-1" role="dialog" aria-labelledby="myModalLabel" aria-hidden="true">
+    <div class="modal-dialog">
+      <div class="modal-content">
+        <div class="modal-header">
+          <button type="button" class="close" data-dismiss="modal" aria-hidden="true">
+            <span class="pficon pficon-close"></span>
+          </button>
+          <h4 class="modal-title" id="myModalLabel">{{imageRepo.metadata.name}}:{{imageTag}}</h4>
+        </div>
+        <div class="modal-body label-tags">
+          <div class="row">
+            <div class="image-icon col-xs-4 col-sm-4 col-md-4">
+              <span class="{{imageRepo | iconClass : 'image' : (imageTag + '.iconClass')}}"></span>
+            </div>
+            <div class="col-xs-8 col-sm-8 col-md-8">
+              <p ng-if="imageRepo | annotation : (imageTag + '.description')">{{imageRepo | annotation : (imageTag + '.description')}}</p>
+              <p>
+                <div ng-if="imageRepo | provider">
+                  <label style="margin-right: 5px;">Provider:</label>{{imageRepo | provider}}
```

Comment:
this should be pulling the provider annotation based on the tag, `{{imageRepo | annotation : (imageTag + '.provider')}}`  and if it doesnt exist fallback to showing the namespace the imageRepo is in